flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_NavRadioPage.js

```diff
@@ -143,14 +148,18 @@ class CDUNavRadioPage {
                 const numValue = parseFloat(value);
                 if (!isFinite(numValue) && value.length >= 2 && value.length <= 3) {
                     mcdu.getOrSelectNDBsByIdent(value, (navaids) => {
-                        mcdu.adf1FreqIsPilotEntered = false;
-                        mcdu.adf1IdIsPilotEntered = true;
-                        mcdu.adf1IdPilotValue = value;
-                        mcdu.adf1Frequency = navaids.infos.frequencyMHz;
-                        mcdu.radioNav.setADFActiveFrequency(1, mcdu.adf1Frequency);
-                        mcdu.requestCall(() => {
-                            CDUNavRadioPage.ShowPage(mcdu);
-                        });
+                        if (navaids) {
+                            mcdu.adf1FreqIsPilotEntered = false;
+                            mcdu.adf1IdIsPilotEntered = true;
+                            mcdu.adf1IdPilotValue = value;
+                            mcdu.adf1Frequency = navaids.infos.frequencyMHz;
+                            mcdu.radioNav.setADFActiveFrequency(1, mcdu.adf1Frequency);
+                            mcdu.requestCall(() => {
+                                CDUNavRadioPage.ShowPage(mcdu);
+                            });
+                        } else {
+                            mcdu.addNewMessage(NXSystemMessages.notInDatabase);
+                        }
```

Comment:
Took the liberty of adding checks here. There may be some 3-letter idents not in the database.